packages/devp2p/src/protocol/eth.ts

```diff
@@ -47,29 +53,33 @@ export class ETH extends Protocol {
   static eth65 = { name: 'eth', version: 65, length: 17, constructor: ETH }
   static eth66 = { name: 'eth', version: 66, length: 17, constructor: ETH }
 
+  
+
   _handleMessage(code: ETH.MESSAGE_CODES, data: any) {
     const payload = arrToBufArr(RLP.decode(bufArrToArr(data)))
     const messageName = this.getMsgPrefix(code)
-    const debugMsg = `Received ${messageName} message from ${this._peer._socket.remoteAddress}:${this._peer._socket.remotePort}`
+    const debugMsg = this.DEBUG ?  `Received ${messageName} message from ${this._peer._socket.remoteAddress}:${this._peer._socket.remotePort}` : undefined;
```

Comment:
Let's move this inside the `if (code... && this.DEBUG)` code section and then you can remove the `this.DEBUG ? ... : undefined` and only create this debug message when the DEBUG flag is iset.

Reply:
That was what I did initially, but then project won't build then because debugMsg is not defined for all the other conditions, but maybe I'm just no doing this right. 
Logically it's fine since all the conditions will only run when var is defined, but I guess linter is just going crazy about that.

Reply:
Ah, didn't catch that. In that case, this approach is good.